add alert for environment settings in docs (#17410)
getsentry/sentry-docs: docs/organization/integrations/coding-agents/claude/index.mdx

2. In Sentry, navigate to **Settings > Integrations** and search for **Claude Agent**.

3. Enter your API key. If your workspace isn't "default" or you want to use a specific environment, update those fields here, then save.
<Alert level="warning">

> If you choose to connect your own Claude environment, it must allow the agent to reach GitHub in order for it to push a branch with the generated fix. In your Claude workspace environment settings, either:

Reviewer: ```suggestion
If you connect your own Claude environment, it needs access to GitHub so the agent can push a branch with the generated fix.

 In your Claude workspace environment settings, either:
```
